exporter/azure_monitor/azure_monitor.go

```diff
@@ -67,10 +68,10 @@ func (exporter *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {
 		}

```

Comment:
You need to be able to give the user to pass in Options in the constructor.

Reply:
Why does the user need to pass in Options. It is not clear to me. I can make two functions where the user could pass in the iKey or Options for NewAzureTraceExporter. I changed the function parameters because I wanted to make something simpler for the developer calling it.

Reply:
Why even have an Options data structure if the user doesn't have the freedom of modifying the values if they wish?